pegasus/sites.v3/code.org/public/css/census_map.css

```diff
@@ -74,3 +74,62 @@
 .blue {
   background: #6699FF;
 }
+
+.yellow {
+  background: #FFFF00;
+}
+
+.grey {
+  background: #D3D3D3;
+}
+
+.census-info-window h4 {
+  margin-top: 4px;
+  margin-bottom: 4px;
+  text-align: center;
+}
+
+.census-info-window hr {
+  border-color: blue;
+  background-color: blue;
+  margin: 2px;
+}
+
+.census-info-window .census-message {
+  text-align: center;
+}
+
+.census-info-window .button-container {
+  width: 280px;
+  margin: 0 auto;
+}
+
+.census-info-window a {
+  text-decoration: none;
+}
+
+.census-info-window .button {
+  display: inline-block;
+  width: 126px;
+  height: 61px;
+  vertical-align: middle;
+  word-wrap: break-word;
+  margin: 5px;
+  color: white;
+  background-color:orange;
+  border-color: orange;
+  border-style: solid;
+  border-width: 1px;
+}
+
+.census-info-window .button:hover {
+  color: orange;
+  background-color: white;
+}
+
+.census-info-window .button-text {
```

Comment:
If you're hitting the point where this would really benefit from SASS, a quick solution is that you can move this to a .scss file in /shared.